### PR TITLE
Add python files for selection sort and insertion sort

### PR DESCRIPTION
**Pull request by lakshyaTaragi**

### CODE REVIEW

File: selectionSort/selectionSort.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+          if arr[minim] > arr[j]:
+             minim = j
+       arr[i], arr[minim] = arr[minim], arr[i]
+    for j in range(len(arr)):
+        print(arr[j])
+    
+    
+selection_sort([5,7,9,0,4])
+print('')
+selection_sort(['i','n','s','e','r','t','i','o','n'])
```
Comment (Matrix278) [inline selectionSort/selectionSort.py:13]: @lakshyaTaragi Why is this line here along with the newline? Maybe we could delete that?

Comment (lakshyaTaragi) [reply selectionSort/selectionSort.py:13]: @Matrix278 Yes, I forgot it there. I've deleted it.

**Comment by Matrix278** [comment]
Thank you for contribution
